phalcon/Forms/Form.zep

```diff
@@ -816,4 +820,109 @@ class Form extends Injectable implements \Countable, \Iterator
     {
         return isset this->elementsIndexed[this->position];
     }
+    
+    /**
+    * Get the value of an attribute
+    *
+    * @throw \InvalidArgumentException  if the attribute does not exists
+    * @return mixed value of attribute
+    */
+    public function getAttribute(string name)
+    {
+        var attribute;
```

Comment:
This can change now to:

```zep
if this->attributes->has(name) {
    return this->attributes->get(name);
}

throw new ....
```

Reply:
Try to get exceptions and edge-cases out of the way first:

```zephir
if !this->attributes->has(name) {
    throw new ....
}

return this->attributes->get(name);
```